crates/nu-command/src/filesystem/open.rs

```diff
@@ -35,6 +36,7 @@ impl Command for Open {
 
     fn signature(&self) -> nu_protocol::Signature {
         Signature::build("open")
+            .input_output_types(vec![(Type::String, Type::Nothing)])
```

Comment:
This should always output `Type::Any` and has currently two variants in that it accepts either path like input or nothing.